model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -159,4 +163,94 @@ class OrtResultTest : WordSpec({
             ortResult.dependencyNavigator should beInstanceOf<DependencyGraphNavigator>()
         }
     }
+
+    "getRuleViolations" should {
+        "return all violations by default" {
+            val ortResult = OrtResult.EMPTY.copy(
+                repository = Repository.EMPTY.copy(
+                    config = RepositoryConfiguration(
+                        resolutions = Resolutions(
+                            ruleViolations = listOf(
+                                RuleViolationResolution(
+                                    "Rule violation message",
+                                    RuleViolationResolutionReason.EXAMPLE_OF_EXCEPTION,
+                                    "comment"
+                                )
+                            )
+                        )
+                    )
+                ),
+                evaluator = EvaluatorRun(
+                    violations = listOf(
+                        RuleViolation(
+                            rule = "rule id",
+                            pkg = Identifier("Maven", "org.ossreviewtoolkit", "resolved-violation", "0.8.15"),
+                            license = null,
+                            licenseSource = null,
+                            severity = Severity.HINT,
+                            message = "Rule violation message",
+                            howToFix = "how to fix message"
+                        )
+                    )
+                )
+            )
+
+            val violations = ortResult.getRuleViolations()
+
+            violations should haveSize(1)
+        }
+
+        "not return any resolved rule violations" {
```

Comment:
consider naming it: "drop resolved rule violations if omitDropped is true".
The test cases above could have similar name with the boolean swapped.